examples/calculate_classify_ndvi.py

```diff
@@ -0,0 +1,149 @@
+from matplotlib.colors import ListedColormap
+import earthpy as et
+import earthpy.spatial as es
+import earthpy.plot as ep
+
+# Get data and set your home working directory
+data = et.data.get_data("cold-springs-fire")
+
+
+###############################################################################
+# Import Example Data
+# -------------------
+#
+# To get started, make sure your directory is set. Create a stack from all of the
+# Landsat .tif files (one per band).
+
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
+
+# Stack the Landsat 8 bands
+# This creates a numpy array with each "layer" representing a single band
+landsat_path = glob(
+    "data/cold-springs-fire/landsat_collect/LC080340322016072301T1-SC20180214145802/crop/*band*.tif"
+)
+landsat_path.sort()
+arr_st, meta = es.stack(landsat_path)
+
+
+###############################################################################
+# Calculate Normalized Difference Vegetation Index (NDVI)
+# -------------------------------------------------------
+
+# You can calculate NDVI for your dataset using the
+# ``normalized_diff`` function from the ``earthpy.spatial`` module.
+# Math will be calculated (b1-b2) / (b1 + b2).
+
+# Landsat 8 red band is band 4 at [3]
+# Landsat 8 near-infrared band is band 5 at [4]
+ndvi = es.normalized_diff(arr_st[4], arr_st[3])
+
+
+###############################################################################
+# Plot NDVI With Colorbar Legend of Continuous Values
+# ----------------------------------------------------
+
+# You can plot NDVI with a colorbar legend of continuous values using the
+# ``colorbar`` function from the ``earthpy.plot`` module.
+
+fig, ax = plt.subplots(figsize=(10, 10))
+
+im = ax.imshow(ndvi, cmap="RdYlGn", vmin=-1, vmax=1)
+
+ep.colorbar(im)
+
+ax.set(title="Landsat 8 - Normalized Difference Vegetation Index (NDVI)")
+ax.set_axis_off()
+plt.show()
+
+
+###############################################################################
+# Classify NDVI
+# -------------
+
+# Next, you can classify the NDVI to categorize the results into useful classes.
+# Values under 0 will be classified together as no vegetation. Additional classes
+# will be created for bare area # and low, moderate, and high vegetation areas.
+
+# Create classes and apply to NDVI results
+ndvi_class_bins = [-np.inf, 0, 0.1, 0.25, 0.4, np.inf]
+ndvi_landsat_class = np.digitize(ndvi, ndvi_class_bins)
+
+# Define class names
+ndvi_cat_names = [
+    "No Vegetation",
+    "Bare Area",
+    "Low Vegetation",
+    "Moderate Vegetation",
+    "High Vegetation",
+]
+
+
+###############################################################################
+# Plot Classified NDVI With Categorical Legend
+# --------------------------------------------
+
+# You can plot the classified NDVI with a categorical legend using the
+# ``draw_legend`` function from the ``earthpy.plot`` module.
+
+# Define color map
+nbr_colors = ["gray", "y", "yellowgreen", "g", "darkgreen"]
```

Comment:
This is good. i think we could use plot_bands here too but i think for now we can leave this as is. i have to figure out how to grab the image object from plot bands which shouldn't be super hard... but maybe super complicated for a user to see.

Reply:
OK will leave as is for now.